Pathfinder/Attribute/EventAttribute.cs

```diff
@@ -1,11 +1,23 @@
 ﻿using System;
+using static Pathfinder.Event.EventManager;
+
 namespace Pathfinder.Attribute
 {
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
     public class EventAttribute : AbstractPathfinderAttribute
     {
         public string DebugName;
-        public int Priority;
+        public int? Priority;
```

Comment:
Not a valid Attribute Parameter type (as Attribute Parameters need to be constant), breaks compile in `Pathfinder/Internal/Gui/ModExtensionsUI.cs`. Should probably stay as `int`.